sunpy/net/scraper.py

```diff
@@ -497,6 +438,5 @@ def get_timerange_from_exdict(exdict):
             tdelta = 366*TIME_QUANTITIES['day']
         else:
             tdelta = 365*TIME_QUANTITIES['day']
-
-    endTime = startTime + tdelta - TIME_QUANTITIES['millisecond']
+    endTime = startTime + tdelta
```

Comment:
Just to check, if we wanted to make the scraper end its time range back at the previous day (like it is now), is this the only change we would need to undo?

Reply:
yes!